plugin-localization/src/main/java/com/mapbox/mapboxsdk/plugins/localization/LocalizationPlugin.java

```diff
@@ -181,7 +186,10 @@ public void setCameraToLocaleCountry() {
    * @since 0.1.0
    */
   public void setCameraToLocaleCountry(Locale locale) {
-    setCameraToLocaleCountry(checkMapLocalNonNull(locale));
+    MapLocale mapLocale = MapLocale.getMapLocale(locale);
+    if (mapLocale != null) {
+      setCameraToLocaleCountry(mapLocale);
+    }
```

Comment:
Worth adding a log as in https://github.com/mapbox/mapbox-plugins-android/pull/689/files#diff-1bf159640ccc4e006c457ee997362bf9R112?

Reply:
nice catch!